site/app/controllers/GlobalController.php

```diff
@@ -364,7 +364,7 @@ public function header() {
         //else if(...){}
         //more Holidays go here!
 
-        return $this->core->getOutput()->renderTemplate('Global', 'header', $breadcrumbs, $wrapper_urls, $sidebar_buttons, $unread_notifications_count, $css->toArray(), $js->toArray(), $duck_img);
+        return $this->core->getOutput()->renderTemplate('Global', 'header', $breadcrumbs, $wrapper_urls, $sidebar_buttons, $unread_notifications_count, $css->toArray(), $js->toArray(), $duck_img, $use_mobile_viewport);
```

Comment:
Let's simplify this by removing passing `$use_mobile_viewport` from Output -> Controller -> View and just have it be the view calls `$this->output->useMobileViewport()` (see below for comment about function changes).